Add files of application layer

LLCCLL opened the pull request:
1. スケジュールを生成する機能を追加しました。

2. 疑問なところを注釈の形で追加しました
（何個は英語で作成しました。
　それらは入力当時UTF-8が適用できなくてですけど、原因不明です。）

3.Google formatter適用されていて、Javadocは適用されていません。

Commit:
Add files of application layer

Add 'Create to-do schedule' function
Some questions about codes applied in annotation form
Google formatter applied
Javadoc not applied

Commit:
Add files of application layer

Add 'Create to-do schedule' function
Some questions about codes applied in annotation form
Google formatter applied
Javadoc not applied

ghost commented:
1. 注釈として表示した疑問なところは、docbaseにまとめて別途アップロードし用としましたが、
作成に時間かかりすぎて、よろしかったらオフライン会で直接質問したいと思います。

2. そしてconflict発生しましたが、
①私のローカルrepositoryのtodoブランチでgit pull upstream master
　＊upstream: https://github.com/btijnb/WebApplication_OJT/
②conflicts発生したファイル編集
③regaseして私のrepositoryにrebase --continue
こうすればいいですか？

@LLCCLL 
--
> 1. 注釈として表示した疑問なところは、docbaseにまとめて別途アップロードし用としましたが、
作成に時間かかりすぎて、よろしかったらオフライン会で直接質問したいと思います。

了解しました。

> 2. そしてconflict発生

すみません。勝手にマスタを編集しちゃいました。
conflictsは明日のオフライン会で状況を見ながら解決しますのでローカルで作業を進めてください。

Commit:
Add files of application layer

Add 'Finish to-do schedule' function
Some annotations are applied that say questions or are just for self-study
Google formatter applied
Javadoc not applied

ghost commented:
@btijnb 
--

>conflictsは明日のオフライン会で状況を見ながら解決しますのでローカルで作業を進めてください。

すみません。pull requestした後、もう一つのコミットをプッシュしました(pull request X)
これについてもオフライン会で助言お願いします。

LLCCLL commented on src/main/java/todo/app/todo/TodoController.java:
```diff
@@ -30,4 +40,28 @@ public String list(Model model) {
+      Model model,
+      RedirectAttributes attributes) {
+    if (bindingResult.hasErrors()) {
+      return list(model); //listメソッドを在実行：　listメソッドに行く(X)→listメソッドの実行結果return(O)?
```
「質問」注釈の内容通り、これの意味が知りたいです。

LLCCLL replied:
「回答」
return list(model): 同じクラスのlistメソッドをreturn (from btijnbさん)

LLCCLL commented on src/main/java/todo/app/todo/TodoController.java:
```diff
@@ -30,4 +40,28 @@ public String list(Model model) {
+
+    attributes.addFlashAttribute(
+        ResultMessages.success()
+            .add(ResultMessage.fromText("Created successfully!"))); //methodにまたmethod?
```
「質問」ResultMessages.success().add　こういった構造が分かりません。

LLCCLL replied:
＊まとめ保留

LLCCLL commented on src/main/java/todo/domain/repository/todo/TodoRepositoryImpl.java:
```diff
@@ -24,7 +24,7 @@ public Todo findOne(String todoId) {
   @Override
   public void create(Todo todo) {
-    TODO_MAP.put(todo.getTodoId(), todo);
+    TODO_MAP.put(todo.getTodoId(), todo); //Why doesn't it retrun anything?
```
「質問」
実行するスケジュールを生成する機能を実現するために、
TodoControllerがTodoServiceを呼んで、TodoRepositoryの中のTODO_MAP.putを実行する構造ですけど、
他の機能のコードと違って、これはなぜreturnをしないか知りたいです。

LLCCLL commented on src/main/java/todo/domain/service/todo/TodoServiceImpl.java:
```diff
@@ -65,7 +65,7 @@ public Todo create(Todo todo) {
     todoRepository.create(todo);
 
-    return todo;
+    return todo; //where is it going after returned?
```
「質問」
todoRepository.create(todo)を実行して得たtodoオブジェクトは、
どこに行くか分かりません。
TodoControllerでは行方を確認できませんでした。

Commit:
Add files of application layer

Add 'Delete to-do schedule' function
Some annotations are applied that say questions or are just for self-study
Google formatter applied
Javadoc not applied

Commit:
Transfer CSS source cords into a file

Google formatter applied
Javadoc not applied

Commit:
change project dir

Commit:
remove target dir

Commit:
fix conflicts
